includes/admin/load.php

```diff
@@ -224,31 +228,85 @@ function perflab_enqueue_modules_page_scripts() {
 	wp_enqueue_script( 'plugin-install' );
 
 	wp_enqueue_script(
-		'perflab-plugin-management',
-		plugin_dir_url( __FILE__ ) . 'js/perflab-plugin-management.js',
+		'perflab-install-activate-plugins',
+		plugin_dir_url( __FILE__ ) . 'js/perflab-install-activate-plugins.js',
 		array( 'jquery' ),
 		'1.0.0',
 		array(
 			'in_footer' => true,
 			'strategy'  => 'defer',
 		)
 	);
+
+	wp_localize_script(
+		'perflab-install-activate-plugins',
+		'perflab_install_activate_plugins',
+		array(
+			'ajaxurl' => admin_url( 'admin-ajax.php' ),
+			'nonce'   => wp_create_nonce( 'perflab-install-activate-plugins' ),
+		)
```

Comment:
I think it should go through `wp_json_encode()`.

Reply:
Not if using `wp_localize_script()`. Using `wp_json_encode()` is only relevant if using `wp_add_inline_script()` where you have to manually encode. For example:

```php
wp_add_inline_script(
    'perflab-install-activate-plugins',
    sprintf( 'var perflab_install_activate_plugins = %s;', wp_json_encode(
		array(
			'ajaxurl' => admin_url( 'admin-ajax.php' ),
			'nonce'   => wp_create_nonce( 'perflab-install-activate-plugins' ),
		)
    ) ),
    'before'
);
```